src/read/pe/file.rs

```diff
@@ -118,6 +118,11 @@ where
     pub(super) fn section_alignment(&self) -> u64 {
         u64::from(self.nt_headers.optional_header().section_alignment())
     }
+
+    /// Returns the root resource directory table
+    pub fn resource_directory_table(&self) -> Result<ResourceDirectoryTable<'data>> {
+        ResourceDirectoryTable::parse(self)
```

Comment:
This should be similar to the parsing for other directory entries, such as `export_table`. Don't pass `self`; instead pass the required information, and probably do it via a method call on `self.data_directories`.

Reply:
Looks indeed more coherent with other things. Done